src/ontology/edits/oeo-physical.omn

```diff
@@ -1975,9 +1975,9 @@ Class: OEO_00000075
         <http://purl.obolibrary.org/obo/IAO_0000115> "Biogasoline is a portion of matter that has a liquid state of matter and has a gasoline fuel role. It consists of bioethanol, biomethanol and products of these two substances. It is made from vegetable or animal material and thus has a biogenic origin.",
         <http://purl.obolibrary.org/obo/IAO_0000116> "The term gasoline was chosen as this term is more often used in energy modelling and energy statistics. This is a deliberate deviation from the convention of British English spelling in the OEO.",
         <http://purl.obolibrary.org/obo/IAO_0000118> "bioethanol"@en,
-        <http://purl.obolibrary.org/obo/IAO_0000233> "https://github.com/OpenEnergyPlatform/ontology/issues/1028
+        OEO_00020426 "https://github.com/OpenEnergyPlatform/ontology/issues/1028
 https://github.com/OpenEnergyPlatform/ontology/pull/1037",
-        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/439
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/439
```

Comment:
The class `biogasoline` has two term tracker annotations. These could be merged into one.